src/js/GaCesium.js

```diff
@@ -178,3 +181,93 @@ var GaCesium = function(map, gaPermalink, gaLayers, gaGlobalOptions, $q) {
 
 };
 
+var SSECorrector = function() {
+  var params = {};
+  location.search.substr(1).split('&').reduce(function(previous, current) {
+    var splitted = current.split('=');
+    if (splitted.length === 2) {
+      params[splitted[0]] = splitted[1];
+    } else {
+      params[splitted[0]] = true;
+    }
+  });
```

Comment:
It was copied from the app viewer. All this code is temporary so I did not spent time using `gaPermalink`. Do you think I should anyway update the PR to use it?